src/DependencyInjection/HostingBundleConfiguration.php

```diff
@@ -18,8 +18,7 @@ public function getConfigTreeBuilder()
 		$treeBuilder->getRootNode()
 			->children()
 				->enumNode("tier")
-					->isRequired()
-					->values(HostingTier::getAllowedTiers())
+					->values(HostingTier::getAllowedConfigValues())
```

Comment:
If only Symfony 6.2 was already here :D https://symfony.com/blog/new-in-symfony-6-2-improved-enum-support

Reply:
I don’t think, that they support enums in config nodes, even if they are called enumNode. 

It at least is not mentioned in the blog post.